ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__opentelemetry-product-ocp-4.15-lp-interop.yaml

```diff
@@ -38,18 +38,18 @@ tests:
       EO_SUB_SOURCE: redhat-operators
       FIREWATCH_CONFIG: |
         [
-          {"step": "build-logs", "failure_type": "pod_failure", "classification": "OpenTelemetry operator images", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
+          {"step": "build-logs", "failure_type": "pod_failure", "classification": "OpenTelemetry operator images", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
```

Comment:
What is this step "build-logs"

Looking at the scenario steps in the artifacts I don't see a step named build-logs
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-open-telemetry-opentelemetry-operator-main-opentelemetry-product-ocp-4.14-lp-interop-opentelemetry-interop-aws/1718871082028503040/artifacts/opentelemetry-interop-aws/

It looks to me like this rule will never take effect as it will never find a step named build-logs.

Did you see this somewhere in a run that I may have missed?